Components/button.css

```diff
@@ -85,7 +86,7 @@
   align-items: center;
   justify-content: center;
```

Comment:
same use flex properties as helper class. In that way you dont have to repeat the code